bin/zold

```diff
@@ -187,7 +187,7 @@ Available options:"
     Zold::Push.new(wallets: wallets, remotes: remotes, log: log).run(args)
   when 'alias'
     require_relative '../lib/zold/commands/alias'
-    Zold::Alias.new(wallets: wallets, remotes: remotes, log: log).run(args)
+    Zold::Alias.new(wallets: wallets, log: log).run(args)
```

Comment:
@botanicus isn't there a way to hide a command from user? If it is used by users, it would be a frustrating UX to have a command that does nothing. Or maybe the gem isn't released that often, I dunno.

@yegor256 what do you think? ^^

Reply:
@ledestin it raises an exception saying that it’s not implemented. For a project under heavy development I don’t think it’s a major problem. My opinion ...

Reply:
@ledestin is this satisfactory?

Reply:
@botanicus I guess it is